Cargo.toml

```diff
@@ -42,8 +42,8 @@ version = "0.15"
 
 [workspace.dependencies.naga]
 git = "https://github.com/gfx-rs/naga"
-rev = "f0edae8"
-version = "0.11"
+rev = "9742f1616c3e3dd2cc9a5880616fc886c391bb9f"
```

Comment:
We can't upgrade naga in a patch unless naga already backported some changes and did a release.

Reply:
It becomes too much of work to get the bug release, I'll close this PR and wait for the next release, thanks @cwfitzgerald